compose/ccd-next-hearing.yml

```diff
@@ -0,0 +1,28 @@
+---
+version: '2.4'
+
+services:
+  ccd-next-hearing-date-updater:
+    image: "${CCD_NEXT_HEARING_UPDATER_LOCAL-hmctspublic.azurecr.io/}ccd/ccd-next-hearing-date-updater:${CCD_NEXT_HEARING_UPDATER_TAG:-latest}"
+    environment:
+      CCD_DATA_STORE_URL: http://ccd-data-store-api:4452
+      IDAM_S2S_URL: http://service-auth-provider-api:8080
+      IDAM_KEY_NEXT_HEARING_UPDATER: AAAAAAAAAAAAAAAA
+      HEARING_NEXT_DAY_CRON_JOB: true
+      CSV_FILENAME: caseReferences.csv
```

Comment:
I think this should be empty by default for docker? Running in a docker container would be looking for a file inside the container that won't ever exist.

Reply:
SI SI, yes I was not sure about   CSV_FILENAME: caseReferences.csv

Reply:
DONE :)